portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/localization/LocalizationWithWebContent.testcase

```diff
@@ -216,6 +216,72 @@ definition {
 			portletName = "Web Content");
 	}
 
+	@description = "This is a use case for LPS-83319. This test is Set And View Web Content with two translations"
+	@priority = "5"
+	test ViewChangeWebContentLanguage {
+		task ("Check Changeable Default Language setting") {
+			ApplicationsMenuHelper.openApplicationsMenu();
+
+			ApplicationsMenu.gotoPortlet(
+				category = "Configuration",
+				panel = "Control Panel",
+				portlet = "System Settings");
+
+			SystemSettings.gotoConfiguration(
+				configurationCategory = "Web Content",
+				configurationName = "Administration",
+				configurationScope = "System Scope");
+
+			FormFields.enableCheckbox(fieldName = "Changeable Default Language");
+
+			SystemSettings.saveConfiguration();
+		}
+
+		task ("Go To Test Site And Create a Web Content") {
+			ApplicationsMenu.gotoSite(key_site = "Test Site Name");
+
+			ProductMenu.gotoPortlet(
+				category = "Content & Data",
+				portlet = "Web Content");
+
+			JSONWebcontent.addWebContent(
+				content = "WC WebContent Content",
+				groupName = "Test Site Name",
+				title = "WC WebContent Title");
+		}
+
+		task ("Set Web Content with two translations") {
+			WebContentNavigator.openWebContentAdmin(siteURLKey = "test-site-name");
+
+			WebContent.viewTitle(webContentTitle = "WC WebContent Title");
+
+			WebContentNavigator.gotoEditCP(webContentTitle = "WC WebContent Title");
+
+			PortletEntry.changeLocale(locale = "es-ES");
+
+			WebContent.addCP(
+				webContentContent = "WC WebContent Content Edit",
+				webContentTitle = "WC WebContent Title Edit");
+
+			PortletEntry.publish();
+		}
+
+		task ("View Web Content with two translations") {
+			WebContentNavigator.gotoEditCP(webContentTitle = "WC WebContent Title");
+
+			Click(locator1 = "Button#CHANGE");
+
+			AssertClick(
```

Comment:
As mentioned on [LPS-83319](https://issues.liferay.com/browse/LPS-83319). I think the steps should be change language to default instead of es_ES. We can have a talk about this tomorrow.